scripts/pylint_extensions_test.py

```diff
@@ -36,6 +36,7 @@
 import astroid  # isort:skip
 import pylint_extensions  # isort:skip
 from pylint import testutils  # isort:skip
+from pylint.lint import PyLinter  # isort:skip  # pylint: disable=import-only-modules
```

Comment:
Please do not import classes, import only modules.

Reply:
Done